doc/man1/openssl-enc.pod.in

```diff
@@ -451,6 +451,13 @@ The B<-list> option was added in OpenSSL 1.1.1e.
 
 The B<-ciphers> and B<-engine> options were deprecated in OpenSSL 3.0.
 
+Since OpenSSL 3.0 using the B<-S> option makes the encrypted file to not
+include the salt into the encrypted file and during the decryption the
+salt is no longer expected to be present in the file. To decrypt files
+encrypted on OpenSSL 1.1.1 with B<-S>, just do not use the B<-S> option.
+To produce files that can be decrypted on OpenSSL 1.1.1 do not use the
+B<-S> option, the salt will be generated randomly.
+
```

Comment:
What about:

> OpenSSL 3.0 changed the way the B<-S> operates to not include the specified salt in the encrypted file.  This prevents decryption using OpenSSL 1.1.1.  To produce files that can be decrypted using OpenSSL 1.1.1 do not use the
B<-S> option with OpenSSL 3.0.  In this case the salt will be generated randomly.

I think this should be in the NOTES section not the HISTORY.

Reply:
I've partially used your proposal also moved to NOTES.